src/pages/Dashboard/components/ReportTable/TableRow/TableRow.tsx

```diff
@@ -15,21 +15,20 @@ export interface TableRowProps {
   projections: Projection[];
 }
 
+export const skillBackgrounds: { [key: string]: string } = {
```

Comment:
should these be moved to the src/components to be shared across the website?

Reply:
i removed a duplicate of this from RoleCard but keeping it in TableRow for now as i'm not sure how skill background colours will be handled in the future (once a user can add many skills)